client/src/app/service/user.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class UserService {
+   private currentUserSubject: BehaviorSubject<any>;
+
+  constructor(private router: Router,
+              private http: HttpClient) {
+         this.currentUserSubject = new BehaviorSubject({});
```

Comment:
format lại khoảng cách dòng của toàn bộ file nhé

Reply:
done